clusterloader2/pkg/measurement/util/kubelet/kubelet.go

```diff
@@ -0,0 +1,144 @@
+/*
```

Comment:
Same comment here - this is pretty independent - can we move this one to a separate PR?

Reply:
Utils for kubelet and kubemark moved to separate PRs - #243 #244 .